code/web/src/modules/user/ProfilePicture.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { logout } from './api/actions'
+
+
+class ProfilePicture extends Component {
+  constructor() {
+    super()
+    this.state = {
+      image: '',
+    }
+  }
+
+  componentDidMount() {
+
+  }
+
+  render() {
+    return (
+      <div>
+      Hello
+      </div>
+    )
+  }
+}
+
+function profileState(state) {
+  return {
+    user: state.user
+  }
+}
+
+export default connect(profileState, { logout })(ProfilePicture)
```

Comment:
I don't remember, is our plan to have this component deal with the profile picture and then import it into Profile?